src/FlashtestationRegistry.sol

```diff
@@ -267,17 +293,17 @@ contract FlashtestationRegistry is
         }
     }
 
-    /**
-     * @notice Returns the TD10ReportBody for the quote used to register a given TEE-controlled address
-     * @param teeAddress The TEE-controlled address to get the TD10ReportBody for
-     * @return reportBody The TD10ReportBody for the given TEE-controlled address
-     * @dev this is useful for when both onchain and offchain users want more
-     * information about the registered TEE than just the workloadId
-     */
-    function getReportBody(address teeAddress) public view returns (TD10ReportBody memory) {
-        bytes memory rawQuote = registeredTEEs[teeAddress].rawQuote;
-        require(rawQuote.length > 0, TEEServiceNotRegistered(teeAddress));
-        return QuoteParser.parseV4Quote(rawQuote);
+    function setTDConfigParams(bytes8 xfamMask, bytes8 tdAttributesMask) external onlyOwner {
+        tdConfigParams = TDConfigParams({xfamMask: xfamMask, tdAttributesMask: tdAttributesMask});
+        emit TDConfigParamsUpdated(xfamMask, tdAttributesMask);
+    }
+
+    event TDConfigParamsUpdated(bytes8 xfamMask, bytes8 tdAttributesMask);
+
+    // Checks tdAttributes and XFAM. Currently defaults to Intel's TCBLevels (rather than maintaining a tcbsvn allowlist).
+    function validateTDConfig(TD10ReportBody memory report) public view returns (bool) {
```

Comment:
readability: Please go into more depth in this comment on _why_ we are doing this type of check, and why these particular values were chosen. It is not clear to me what `Currently defaults to Intel's TCBLevels (rather than maintaining a tcbsvn allowlist)` is referring to. What are Intel's TCBLevels? What is a tcbsvn allowlist and why aren't we using one?